content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -1,225 +1,176 @@
-
-Should you use variants?
-========================
-
-When should you use variants?
------------------------------
-
-Using variants has the following impacts:
-
-- **eCommerce**: in your online shop, the customer will only see
-  product templates in the catalog page. Once the visitor click on
-  such a product, he will have options to choose amongst the
-  variants (colors, sizes, …)
+Configuration
+=============
 
-- **Manufacturing**: Using variants allows to define only one bill of
-  material for a product template and slight variations for some of
-  the variants. Example: instead of creating a Bill of Material for
-  "T-shirt, Red, S", you create a bill of material for "T-shirt"
-  and add some lines that are specific to the dimension S, and
-  other lines specific to the color Red.
+Activating the variant feature
+------------------------------
 
-- **Pricing**: The default price of a product is computed using the
-  price of the product template and add the optional extra price on
-  each dimension of the variant. This way, variant prices are
-  easier to maintain since you don't have to set the price for
-  every variant. However, it's possible to create pricelist rules
-  to fix price per variants too.
+Before using product variants, the setting needs to be activated. To do so, you must go to the
+Sales app. Select :menuselection:`Sales --> Configuration --> Settings`, locate the **Products
+Variants** line via scrolling or the search bar at the top of the page.
 
-When should you avoid using variants?
--------------------------------------
+.. image:: variants/activating-variants-setting.png
+   :align: center
+   :alt: Activating product variants
 
-Using variants may add a level of complexity on the way you use Odoo.
-You should consider using variants only if you need it to reduce the
-complexity of managing lots of products that are similars.
+Creating attributes
+-------------------
 
-As an example, importing your initial product catalog is more complex if
-you use variants. You can't just import a list of products, you must
-import product templates and all their related variations.
+Attributes need to be created before product variants can be set up in your database. Attributes
+can be accessed via :menuselection:`Configuration --> Attributes`. To create a new attribute,
+click **Create**. First, choose an attribute name, such as “color” or “size.”
 
-In addition to that, you should also carefully select the dimensions
-that you manage as separate product templates and those as variants. As
-an example, a company having these products:
+.. image:: variants/attribute-creation.png
+   :align: center
+   :alt: Attribute creation window
 
-- Quality: T-Shirts, Polos, Shirts
+Then, select a display type from *Radio*, *Select*, and *Color*, which determines how your product
+will be shown on your eCommerce page, PoS dashboard and Product Configurator.
 
-- Color: Red, Blue
+- *Radio*: Options are in a bullet style list
```

Comment:
```suggestion
- *Radio*: Options are in a bullet style list.
```
Missing full stop

Reply:
@vpk-odoo I guess you forgot this. It's very minor so I'll approve the PR & move it to the next stage. You will be able to address it when Jon requests changes :)